app/src/test/java/fr/iut_rodez/pathpilot_android_client/util/ValidateFormTest.java

```diff
@@ -0,0 +1,50 @@
+package fr.iut_rodez.pathpilot_android_client.util;
+
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class ValidateFormTest {
```

Comment:
Missing Javadoc comment to describe what the activity is for

Reply:
Fix in `3f381088a74825447a65980cc8bb98e00dd60954`